sender.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+source_file=$1
+stopServer=$2
+host=$3
+statusCode=0
+
+if [ "hostType" == "m" ]
```

Shellcheck (0.10.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)